language/move-compiler/tests/move_check/unit_test/expected_failure_out_of_range_value.unit_test.exp

```diff
@@ -1,9 +1,25 @@
+warning[W10007]: potential issue with attribute value
+  ┌─ tests/move_check/unit_test/expected_failure_out_of_range_value.move:6:24
+  │
+6 │     #[expected_failure(abort_code=18446744073709551614)]
+  │                        ^^^^^^^^^^ -------------------- Replace this value with a constant or add the 'location=...' attribute
+  │                        │           
+  │                        WARNING POTENTIALLY INACCURATE TEST. The value should point to a constant defined in the module that will abort to signify which module will abort. Or the location attribute must be specified.
```

Comment:
Do you have a suggestion for shortening it? I don't think we automatically column fit currently

Reply:
Hmm, well seeing as we get the tip about replacing it with a comment or adding the `location` attribute, how about just warning about the problem, something like:

```
WARNING: passes for abort from any module.
```

It may need the tip to be reworded slightly to make it clear why using a constant helps, something like:

```
Replace value with constant from expected module or add `location=...` attribute.
```